bin/deploy.sh

```diff
@@ -27,6 +27,16 @@ mkdir -p $BIN_DIR/../deployments
 
 [[ -z "$GOVERNANCE" ]] && GOVERNANCE="$ETH_FROM"
 
+DEPLOYMENT_FILE="./../deployments/addresses_$(seth chain).json"
+if [[ -z "$RESUME" ]]; then
```

Comment:
I think you should change it to an equal == 1 here or otherwise change the description. 

Because `RESUME=0` would be the same as `RESUME=1`

Reply:
Maybe we can echo a message here if the addresses got read from a file

Reply:
I'm echoing it here: https://github.com/centrifuge/tinlake-deploy/blob/tinlake-0.3.5/bin/util/env-check.sh#L213. Adding an equals sign is a good idea, will do that!